metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/parser.py

```diff
@@ -48,7 +48,7 @@ def get_upstream_tables(
     native_query_enabled: bool = True,
 ) -> List[resolver.DataPlatformTable]:
     if table.expression is None:
-        logger.debug(table.full_name, "Expression is none")
+        logger.debug("Expression is none for table %s", table.full_name)
```

Comment:
Please, can you use fstring instead of format strings? 
Something like this:
```
logger.debug(f"Expression is none for table {table.full_name}")
```

Reply:
I used above format because argument only get process if debug is enabled